README.md

```diff
@@ -32,22 +32,74 @@ As Administrator:
 
 ### Docker
 
-**NB** - `cht-conf` in a Docker container should only be used if you're familiar with running containers. You will have to copy or mount your config files into the container in order to be able to use the `cht` call to interact with a remote CHT instance.
+CHT Conf can also be run from within a Docker container. This is useful if you are already familiar with Docker and do not wish to configure the various dependencies required for developing CHT apps on your local machine. The necessary dependencies are pre-packaged in the Docker image.
 
-Build and then run the `cht-conf` container:
+#### Building the image
 
-	docker build -t cht-conf:v0 .
-	docker run -d --name cht-conf cht-conf:v0
+Install [Docker](https://www.docker.com/). If you are using Windows, you also need to enable the [Windows Subsystem for Linux (WSL2)](https://learn.microsoft.com/en-us/windows/wsl/install) to perform the following steps.
 
-You now have a container running in the background called `cht-conf`. You can get a shell on this container with:
+Using the terminal (or the WLS shell on Windows: _Start > wsl_), run the following commands to create a new project directory and build the Docker image:
 
-	docker exec -it cht-conf /bin/bash
+```shell
+mkdir -p ~/cht-project
+cd ~/cht-project
+curl https://raw.githubusercontent.com/medic/cht-conf/main/Dockerfile > Dockerfile
+docker build -t cht-conf .
+```
 
-And then you can run the utility with `cht`.  
+#### Using the image
 
-When you are done with the container, you can stop it with:
+The resulting Docker image can be used as a [VSCode Development Container](https://code.visualstudio.com/docs/devcontainers/containers) (easiest) or as a standalone Docker utility.
 
-	docker stop cht-conf
+##### VSCode Development Container
+
+If you want to develop CHT apps with VSCode, you can use the Docker image as a Development Container. This will allow you to use the `cht-conf` utility and its associated tech stack from within VSCode (without needing to install dependencies like NodeJS on your host system).
+
+[Install VSCode](https://code.visualstudio.com/) if you do not have it already.
+
+Using the terminal (or the WLS shell on Windows: _Start > wsl_), run the following commands from within your project directory (created above) to download the `.devcontainer.json` config file, install the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers), and open the project directory in VSCode:
+
+```shell
+curl https://raw.githubusercontent.com/medic/cht-conf/main/devcontainer.config/.devcontainer.json > .devcontainer.json
+code --install-extension ms-vscode-remote.remote-containers
+code -n .
+```
```

Comment:
Minor point to keep the whole "copy paste and it just works!" vibe goin by adding a `cd` and then still adding a `-s` to quiet up `curl`:

```suggestion

```shell
cd ~/cht-project
curl -s https://raw.githubusercontent.com/medic/cht-conf/main/devcontainer.config/.devcontainer.json > .devcontainer.json
code --install-extension ms-vscode-remote.remote-containers
code -n .
```
```